Chapters/05_Monads.md

```diff
@@ -84,19 +84,27 @@ TODO[[ Use Strings everywhere instead of Char]]
 // Monads/ShowResult.scala
 
 def check(
+    // This isn't really an id. Better name?
     id: String,
     end: String,
-    msg: String
+    history: String
 ): Result =
   val result =
     if end == id then
-      Fail(msg + id)
+      Fail(history + id)
     else
-      Success(msg + id)
-  println(s"$end => check($id): $result")
+      Success(history + id)
+  println(s"check($id, $end): $result")
```

Comment:
We should also consider showing `history` as the 3rd parameter here.
Then we completely eliminate the discrepancy between the real 3 argument function, and the false 1 or 2 argument version for displaying.
Further, tracing the history parameter is probably just as valuable as the other 2.